File: tools/calibrate-pipeline/fixtures/ast-diff-trivial-check/good-01-real-migration/input.ts
import { test, expect } from "@playwright/test";

test("login flow", async ({ page }) => {
  await page.goto("https://app.acme.test/login");
  await page.locator("#email").fill("jane@acme.test");
  await page.locator("#password").fill("hunter2");
  await page.locator("button.primary").click();
  await page.waitForTimeout(2000);
  const visible = await page.locator(".dashboard-greeting").isVisible();
  expect(visible).toBe(true);
});

File: tools/calibrate-pipeline/fixtures/ast-diff-trivial-check/bad-04-java-rename-only/input.java
package com.acme.shop.tests;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class LoginTest {

    private WebDriver driver;

    @BeforeEach
    void setUp() {
        driver = new ChromeDriver();
        driver.get("https://app.acme.test/login");
    }

    @AfterEach
    void tearDown() {
        driver.quit();
    }

    @Test
    void validCredentialsLandOnDashboard() throws InterruptedException {
        driver.findElement(By.id("email")).sendKeys("jane@acme.test");
        driver.findElement(By.id("password")).sendKeys("hunter2");
        driver.findElement(By.cssSelector("button.primary")).click();
        Thread.sleep(2000);
        WebElement dashboard = driver.findElement(By.cssSelector(".dashboard-greeting"));
        assertTrue(dashboard.isDisplayed());
    }
}
